lib/l10n/app_en.arb

```diff
@@ -443,6 +443,30 @@
   "@list": {},
   "grid": "Grid",
   "@grid": {},
+  "customizationSettingsTitle": "Customization",
+  "@customizationSettingsTitle": {
+    "description": "Title for the customization settings screen"
+  },
+  "playbackSpeedControlSetting": "Playback Speed Visibility",
+  "@playbackSpeedControlSetting": {},
+  "playbackSpeedControlSettingSubtitle": "Whether the playback speed controls are shown in the player screen menu",
+  "@playbackSpeedControlSettingSubtitle": {},
+  "automatic": "Automatic",
+  "@automatic": {},
+  "shown": "Shown",
+  "@shown": {},
+  "hidden": "Hidden",
+  "@hidden": {},
+  "speed": "Speed",
+  "@speed": {},
+  "reset": "Reset",
+  "@reset": {},
+  "apply": "Apply",
+  "@apply": {},
+  "on": "On",
+  "@on": {},
+  "off": "Off",
+  "@off": {},
```

Comment:
They're not used, right? But I guess they don't really hurt

Reply:
I thought so as well, but I will remove them. If they are reintroduced, they are reintroduced.